### PR TITLE
Cleanup example projects

### PR DESCRIPTION
**Pull request by hellwolf**
## Checklist

- [x] removed dev-playground
- [x] removed react-ts-sdk
- [x] updated example projects' dependencies
- [x] handler.build-examples.yml to support generic build-and-test.sh as build step
- [x] include streaming-call-option
- [x] include nftbillboard-userdata

### CODE REVIEW

**Comment by saflamini** [comment]
1) Added build-and-test.sh file to 2 additional examples
2) Added test cases to 2 additional examples
3) Fixed errors in flowlottery build script

@hellwolf all examples built and ran successfully. Please let me know if you see something I may have missed

**Comment by hellwolf** [comment]
@saflamini looks good!

File: examples/streaming-call-option/artifacts/@openzeppelin/contracts/introspection/ERC165.sol/ERC165.dbg.json [hunk 1]
```diff
@@ -1,4 +1,4 @@
 {
```
Comment (0xdavinchee) [inline examples/streaming-call-option/artifacts/@openzeppelin/contracts/introspection/ERC165.sol/ERC165.dbg.json:1]: I think it's best to add all `artifacts` or any sort of generated files in `.gitignore`.

File: examples/nftbillboard-userdata/build-and-test.sh [hunk 2]
```diff
@@ -0,0 +1,21 @@
+#!/bin/env bash
```
Comment (0xdavinchee) [inline examples/nftbillboard-userdata/build-and-test.sh:1]: I think that a few of these `build-and-test.sh` files are repetitive and can be either replaced by the `testenv-ctl.sh` file in `ethereum-contracts` or if these files are different enough, to have their own `testenv-ctl.sh` file for all example projects-it seems that the projects will all have somewhat similar environment set ups unless there's an example project which exists to illustrate a new way of setting up a test environment (e.g. w/ hardhat network or off a fork, etc.)

File: examples/README.md [hunk 3]
```diff
@@ -1,11 +1,6 @@
-# Example SuperApps
+TODO...
```
Comment (kasparkallas) [inline examples/README.md:1]: TODO

File: examples/streaming-call-option/hardhat.config.js [hunk 4]
```diff
@@ -32,13 +33,18 @@ module.exports = {
...
+  //     url: "http://127.0.0.1",
+  //     network_id: "*",
+  //     port: 8545,
+  // },
+    // rinkeby: {
+    //   url: `${process.env.RINKEBY_ALCHEMY_URL}`,
+    //   gasPrice:  1500000000,
+    //   accounts: [`0x${process.env.RINKEBY_DEPLOYER_PRIVATE_KEY}`]
+    //   },
+    // },
```
Comment (kasparkallas) [inline examples/streaming-call-option/hardhat.config.js:47]: Commented out code

**Review by kasparkallas — APPROVED** [review]
I wonder if the example frond-end apps with MetaMask when hosted on a site like https://codesandbox.io

File: examples/README.md (hunk 3 above)
Comment (saflamini) [reply examples/README.md:1]: will look into https://codesandbox.io - could be interesting to do for your redux sdk example

File: examples/nftbillboard-userdata/build-and-test.sh (hunk 2 above)
Comment (saflamini) [reply examples/nftbillboard-userdata/build-and-test.sh:1]: My one concern here is with using different test environments. I was able to use ganache + truffle within two of the hardhat projects, but may want to use hardhat network specifically for an example project in the future. 

If this is the case, would it still be best to add a` testenv-ctl.sh` file for all example projects?

Comment (hellwolf) [reply examples/nftbillboard-userdata/build-and-test.sh:1]: We can leave as it is for this branch. When we have a new testenv-ctl published, we update the examples to use it.

Comment (saflamini) [reply examples/nftbillboard-userdata/build-and-test.sh:1]: Sounds good

**Review by saflamini — APPROVED** [review]

**Pull request merged**